### PR TITLE
New version: Foxit.PhantomPDF 11.0.1.49938

### PR DESCRIPTION
**Pull request by lychichem**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/28143)

### CODE REVIEW

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Is this the correct link? Did they rebrand to `Foxit PDF Editor`?
The Publisher is also `Foxit Software Inc.`
Think this might be a rogue package that have not been updated because the information is very wrong...
Can you create a new Package with the correct information under `Foxit.PDFEditor` or `Foxit.FoxitPDFEditor`?

File: manifests/f/Foxit/PhantomPDF/11.0.1.49938/Foxit.PhantomPDF.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Automatically updated by the winget bot at 2021/May/28
+PackageIdentifier: Foxit.PhantomPDF
+Publisher: Foxit
+PackageName: PhantomPDF
```
Comment (ItzLevvie) [inline manifests/f/Foxit/PhantomPDF/11.0.1.49938/Foxit.PhantomPDF.yaml:4]:
```suggestion
PackageName: Foxit PDF Editor
```

File: manifests/f/Foxit/PhantomPDF/11.0.1.49938/Foxit.PhantomPDF.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Automatically updated by the winget bot at 2021/May/28
+PackageIdentifier: Foxit.PhantomPDF
+Publisher: Foxit
```
Comment (ItzLevvie) [inline manifests/f/Foxit/PhantomPDF/11.0.1.49938/Foxit.PhantomPDF.yaml:3]:
```suggestion
Publisher: Foxit Software Inc.
```

File: manifests/f/Foxit/PhantomPDF/11.0.1.49938/Foxit.PhantomPDF.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+PackageName: PhantomPDF
+PackageVersion: 11.0.1.49938
+License: 2020 © Foxit Software Incorporated. All rights reserved.
+InstallerType: exe
+Moniker: FoxitPhantomPDF
+Tags:
+- PDF
+- Foxit
```
Comment (ItzLevvie) [inline manifests/f/Foxit/PhantomPDF/11.0.1.49938/Foxit.PhantomPDF.yaml:11]:
```suggestion
Moniker: foxitphantompdf
Tags:
- pdf
- foxit
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**